NOTICE

```diff
@@ -3,3 +3,7 @@ Copyright [2014-2016] The Apache Software Foundation
 
 This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
+
+
+Portions of this software were developed at
```

Comment:
@clebertsuconic can you link to me an example of where you've seen it? Usually NOTICE is only when code is imported via non-SGA, e.g. a compatible OS license where modifications are done.